examples/hello-world/hello_world.ipynb

```diff
@@ -269,19 +270,18 @@
-    "\n",
+    "> 📝 Currently, we do not pickle the function, but rely on Python module references for serializing the configured function. This means that executing a function defined in this notebook (or in the same script calling `run.run`) will probably not work. During packaging, we currently use `git archive` for our remote Slurm executor. You can specify a subpath which will be set as the working directory for the execution. To execute `add_object` function in this example, you need to provide a subpath of `examples/hello-world`, assuming `examples` is at the root of the repository. This will ensure that, during remote execution, imports of the style `from simple.add import add_object` work properly. We are looking to improve this user experience and open to suggestions, so please reach out to us if you have any ideas. We also plan to add `cloudpickle` support for arbitrarily defined functions.\n",
     "\n",
-    "Under the hood, we are using [TorchX](https://pytorch.org/torchx/latest/) to manage the execution. But this is fairly abstracted away from the user and we can potentially support more standlone executor libraries or add custom schedulers for TorchX in the future.\n",
+    "Under the hood, we are using [TorchX](https://pytorch.org/torchx/latest/) to manage the execution. However, this is fairly abstracted away from the user and we can potentially support more standlone executor libraries or add custom schedulers for TorchX in the future.\n",
     "\n",
-    "Let's see the local executor in action. The local executor is the simplest executor we have, and can be intialized without any arguments. Once we have an instance of `run.LocalExecutor`, we can pass it to `run.run` to execute the same configured function on the local executor, which will run the function in a separate process. This is the ease of use provided by NeMo Run. You can configure a function once, and execute it on any supported remote cluster seamlessly. Later, we will also explore `run.Experiment` which allows you to combine and manage multiple runs; giving additional flexibility to the user.\n",
+    "Let's see the local executor in action. The local executor is the simplest executor we have and can be intialized without any arguments. Once we have an instance of `run.LocalExecutor`, we can pass it to `run.run` to execute the same configured function on the local executor, which will run the function in a separate process. This demonstates the ease of use provided by NeMo-Run. You can configure a function once and execute it on any supported remote cluster seamlessly. Later, we will also explore `run.Experiment` which allows you to combine and manage multiple runs, providing additional flexibility to the user.\n",
```

Comment:
fix spelling "demonstrates".

"This demonstrates the ease of use provided by NeMo-Run. You can configure a function once and execute it on any supported remote cluster seamlessly.